tests/ckeditor/ckeditor.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect */
+
+'use strict';
+
+bender.amd.require( 'ckeditor', 'ckeditor-core' );
+
+describe( 'getPluginPath()', function( CKEDITOR, core ) {
```

Comment:
> But eventually this mechanism will be adopted by benderjs-amd

Just to underline this, this feature has been coded with the CKEditor 5 dev environment in mind, exclusively, and is proposed in the ckeditor5 plugin for bender right because of this. It is optimized for our needs and the way we code tests.